### PR TITLE
CNV-9053 KubeMacPool enabled by default

### PR DESCRIPTION
**Pull request by aburdenthehand**
Rewording and rearranging the KubeMacPool content now that it is enabled by default.

[CNV-9053](https://issues.redhat.com/browse/CNV-9053)

[Preview build](https://deploy-preview-31204--osdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/vm_networking/virt-using-mac-address-pool-for-vms.html)

### CODE REVIEW

File: modules/virt-about-kubemacpool.adoc [hunk 1]
```diff
@@ -15,6 +15,5 @@ Virtual machine instances created from that virtual machine retain the assigned
 KubeMacPool does not handle virtual machine instances created independently from a virtual machine.
 ====
 
-KubeMacPool is disabled by default.
-Enable a MAC address pool for a namespace by applying the KubeMacPool label to the namespace.
-
+KubeMacPool is enabled by default when you install {VirtProductName}.
+You can disable a MAC address pool for a namespace by removing the KubeMacPool label to the namespace. You can then re-enable it by adding the KubeMacPool label.
```
Comment (RamLavi) [inline modules/virt-about-kubemacpool.adoc:19]:
```suggestion
You can disable a MAC address pool for a namespace by adding the KubeMacPool disabling label to the namespace. You can then re-enable it by removing this label.
```

**Review by RamLavi — CHANGES_REQUESTED** [review]
Hi,
It is important to emphasis that now that Kubemacpool is enabled by default, 
disabling KMP now happens when you **add a disabling label**.
similarly, re-enabling happens when you remove this disabling label.

Please go over the document and change appropriately

File: modules/virt-about-kubemacpool.adoc (hunk 1 above)
Comment (aburdenthehand) [reply modules/virt-about-kubemacpool.adoc]: @Ram - Thanks for that. Have updated the docs.

File: modules/virt-about-kubemacpool.adoc [hunk 2]
```diff
@@ -15,6 +15,5 @@ Virtual machine instances created from that virtual machine retain the assigned
 KubeMacPool does not handle virtual machine instances created independently from a virtual machine.
 ====
 
-KubeMacPool is disabled by default.
-Enable a MAC address pool for a namespace by applying the KubeMacPool label to the namespace.
-
+KubeMacPool is enabled by default when you install {VirtProductName}.
+You can disable a MAC address pool for a namespace by adding the KubeMacPool `ignore` label to the namespace. Re-enable KubeMacPool for the namespace by removing the label.
```
Comment (RamLavi) [inline modules/virt-about-kubemacpool.adoc:19]: I think it may be better to call it "disabling label". but I'm not totally sure about that. what do you think?

```suggestion
You can disable a MAC address pool for a namespace by adding the KubeMacPool disabling label to the namespace. Re-enable KubeMacPool for the namespace by removing the label.
```

Comment (RamLavi) [reply modules/virt-about-kubemacpool.adoc:19]: @dankenigsberg what do you think?

Comment (dankenigsberg) [reply modules/virt-about-kubemacpool.adoc:19]: I don't know... I prefer to be explicit as `ignore` is. However, I also prefer to be precise, and the precise label is `mutatevirtualmachines.kubemacpool.io=ignore`, not just `ignore`.

File: modules/virt-enabling-mac-address-pool-for-namespace-cli.adoc [hunk 3]
```diff
@@ -5,15 +5,15 @@
 [id="virt-enabling-mac-address-pool-for-namespace-cli_{context}"]
 = Enabling a MAC address pool for a namespace in the CLI
 
-Enable a MAC address pool for virtual machines in a namespace by applying the `mutatevirtualmachines.kubemacpool.io=allocate` label to the namespace.
+If you disabled KubeMacPool for a namespace and want to re-enable it, remove the `mutatevirtualmachines.kubemacpool.io=ignore` label from the namespace.
```
Comment (dankenigsberg) [inline modules/virt-enabling-mac-address-pool-for-namespace-cli.adoc:8]: Since upgrade cluster could still have `allocate`, I think that it would be worthwhile to add 
```suggestion
If you disabled KubeMacPool for a namespace and want to re-enable it, remove the `mutatevirtualmachines.kubemacpool.io=ignore` label from the namespace. Former versions suggested to set `mutatevirtualmachines.kubemacpool.io=allocate`. This is still supported but redundant.
```

File: modules/virt-about-kubemacpool.adoc (hunk 2 above)
Comment (RamLavi) [reply modules/virt-about-kubemacpool.adoc:19]: then I think it should look like this:
```suggestion
You can disable a MAC address pool for a namespace by adding the KubeMacPool `mutatevirtualmachines.kubemacpool.io=ignore` label to the namespace. Re-enable KubeMacPool for the namespace by removing the label.
```

Comment (dankenigsberg) [reply modules/virt-about-kubemacpool.adoc:19]: maybe even a shorter version
```suggestion
You can disable a MAC address pool for a namespace by adding the `mutatevirtualmachines.kubemacpool.io=ignore` label to the namespace. Re-enable KubeMacPool for the namespace by removing the label.
```

Comment (aburdenthehand) [reply modules/virt-about-kubemacpool.adoc:19]: Updated as suggested. Thanks for the input, folks

File: modules/virt-enabling-mac-address-pool-for-namespace-cli.adoc (hunk 3 above)
Comment (aburdenthehand) [reply modules/virt-enabling-mac-address-pool-for-namespace-cli.adoc:8]: Good point. I've modified this a bit and added it as a note to the intro para:
`[NOTE]
====
Earlier versions of {VirtProductName} used the label `mutatevirtualmachines.kubemacpool.io=allocate` to enable KubeMacPool for a namespace. This is still supported but redundant as KubeMacPool is now enabled by default.
====`

File: modules/virt-about-kubemacpool.adoc (hunk 2 above)
Comment (RamLavi) [reply modules/virt-about-kubemacpool.adoc:19]: can you update the preview link?

Comment (aburdenthehand) [reply modules/virt-about-kubemacpool.adoc]: Just pushed the commit. The preview link should update in ~5 minutes

**Review by yossisegev — COMMENTED** [review]
/LGTM

**Review by RamLavi — APPROVED** [review]
/lgtm

File: modules/virt-disabling-mac-address-pool-for-namespace-cli.adoc [hunk 4]
```diff
@@ -5,15 +5,15 @@
...
 
-Disable a MAC address pool for virtual machines in a namespace by removing the `mutatevirtualmachines.kubemacpool.io` label.
+Disable a MAC address pool for virtual machines in a namespace by adding the `mutatevirtualmachines.kubemacpool.io=ignore` label to the namespace.
 
 .Procedure
 
-* Remove the KubeMacPool label from the namespace.
-The following example removes the KubeMacPool label from two namespaces, `<namespace1>` and `<namespace2>`:
+* Add the `mutatevirtualmachines.kubemacpool.io=ignore` label to the namespace.
+The following example adds the KubeMacPool label to two namespaces, `<namespace1>` and `<namespace2>`:
```
Comment (sjhala-ccs) [inline modules/virt-disabling-mac-address-pool-for-namespace-cli.adoc:13]: To avoid confusion between KubeMacPool (the service) and the actual label, I would suggest rewording this as "The following example disables KubeMacPool for two namespaces, `<namespace1>` and `<namespace2>`:"

File: modules/virt-enabling-mac-address-pool-for-namespace-cli.adoc [hunk 5]
```diff
@@ -5,15 +5,20 @@
...
+[NOTE]
+====
+Earlier versions of {VirtProductName} used the label `mutatevirtualmachines.kubemacpool.io=allocate` to enable KubeMacPool for a namespace. This is still supported but redundant as KubeMacPool is now enabled by default.
+====
 
 .Procedure
 
-* Add the KubeMacPool label to the namespace.
-The following example adds the KubeMacPool label to two namespaces, `<namespace1>` and `<namespace2>`:
+* Remove the KubeMacPool label from the namespace.
```
Comment (sjhala-ccs) [inline modules/virt-enabling-mac-address-pool-for-namespace-cli.adoc:17]: Same as above. Suggestion to reword this as "Remove the `mutatevirtualmachines.kubemacpool.io=ignore` label from the namespace. The following example re-enables KubeMacPool  for two namespaces, `<namespace1>` and `<namespace2>`:

File: modules/virt-enabling-mac-address-pool-for-namespace-cli.adoc [hunk 6]
```diff
@@ -5,15 +5,20 @@
 [id="virt-enabling-mac-address-pool-for-namespace-cli_{context}"]
 = Enabling a MAC address pool for a namespace in the CLI
```
Comment (sjhala-ccs) [inline modules/virt-enabling-mac-address-pool-for-namespace-cli.adoc:6]: Would it make sense to change the title to `Re-enabling a MAC address pool for a namespace in the CLI`?

**Review by sjhala-ccs — COMMENTED** [review]
A couple of nits, otherwise LGTM
